package.json

```diff
@@ -16,16 +16,19 @@
     "funit": "PUPPETEER_PRODUCT=firefox node test/test.js",
     "debug-unit": "node --inspect-brk test/test.js",
     "test-doclint": "node utils/doclint/check_public_api/test/test.js && node utils/doclint/preprocessor/test.js",
-    "test": "npm run lint --silent && npm run coverage && npm run test-doclint && npm run test-types && node utils/testrunner/test/test.js",
+    "test": "npm run tsc && npm run lint --silent && npm run coverage && npm run test-doclint && npm run test-types && node utils/testrunner/test/test.js",
+    "prepublishOnly": "npm run tsc",
+    "dev-install": "npm run tsc && node install.js",
     "install": "node install.js",
     "lint": "([ \"$CI\" = true ] && eslint --quiet -f codeframe . || eslint .) && npm run tsc && npm run doc",
     "doc": "node utils/doclint/cli.js",
     "coverage": "cross-env COVERAGE=true npm run unit",
-    "tsc": "tsc --version && tsc -p .",
+    "tsc": "tsc -p . && cp src/protocol.d.ts lib/ && cp src/externs.d.ts lib/",
```

Comment:
Why remove the `tsc --version`?

Reply:
I accidentally left it in from debugging, but thinking about it it's probably useful so I'll add it back.